### DISCUSSION TITLE
How to increase MaxActiveRunsPerDag effectively

### DISCUSSION THREAD

**Discussion by Fthayy** [apache/airflow#29060] [Q&A]
I want to increase max_active_runs_per_dag but it causes to DAG slow down. How can I make the DAG not slow down when I increase max_active_runs_per_dag

**Comment by potiuk** [comment]
The question is too generic. You have to be more explicit what you mean by slow-down. I think you need to look for reaons of your slow-down and analyse what changed in your circumstance if you observe slow-downs. There are mutliple impacting factors - memory, database, your filesystem, number of CPUS - generally any resources you have might limit the performance. 

You need to spend a bit more time on first of all looking and describing your deployment and secondly on analysing what kind of behaviour change you see and making some investigation first if you want anyone to effectively help you to solve your problem.
